bigtable/api/HelloWorld/HelloWorld.cs

```diff
@@ -174,7 +178,7 @@ private static void DoHelloWorld()
 
                 // Read a specific row. Apply filter to return latest only cell value accross entire row.
                 Row rowRead = bigtableClient.ReadRow(
-                    tableName, rowKey: rowKeyPrefix + rowIndex, filter: RowFilters.CellsPerRowLimit(1));
+                    tableName, rowKey: rowKeyPrefix + rowIndex, filter: filter);
```

Comment:
Now that this is called filter, are you able to just pass "filter" as the arg instead of "filter: filter"
For below too